package/ios/CameraView.swift

```diff
@@ -45,6 +45,14 @@ public final class CameraView: UIView, CameraSessionDelegate {
   // other props
   @objc var isActive = false
   @objc var torch = "off"
+  @objc var torchLevel: NSNumber = 0.7
+  @objc var torchDelay: NSNumber = 0
+  @objc var torchDuration: NSNumber = 0
+  @objc var backgroundLevel: NSNumber = 0.0
+  @objc var backgroundDelay: NSNumber = 0
+  @objc var backgroundDuration: NSNumber = 0
+  @objc var enableBackgroundTorch = false
```

Comment:
I think we should probably pull enableBackgroundTorch out.  It should be the responsibility of the code setting up the Camera props to just set all backgroundTorch values to 0 when it's undesired.